DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/EditCourseDetails.cshtml

```diff
@@ -0,0 +1,192 @@
+﻿@inject IConfiguration configuration
+@using DigitalLearningSolutions.Web.Extensions
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewComponents
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@using Microsoft.AspNetCore.Mvc.TagHelpers
+@using Microsoft.Extensions.Configuration
+@model EditCourseDetailsViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = "Edit course details";
+  ViewData["Application"] = DlsSubApplication.TrackingSystem.Name;
+  ViewData["HeaderPath"] = DlsSubApplication.TrackingSystem.HeaderPath;
+  ViewData["HeaderPathName"] = DlsSubApplication.TrackingSystem.HeaderPathName;
+  var cancelLinkData = Html.GetRouteValues();
+  var customisationId = Html.GetRouteValueByKey("customisationId");
+  const string passwordId = "conditional-password";
+  const string emailId = "conditional-email";
+  const string completionCriteriaId = "conditional-completion-criteria";
+  var passwordCss = "nhsuk-checkboxes__conditional"
+                    + (Model.PasswordProtected ? "" : " nhsuk-checkboxes__conditional--hidden")
+                    + (errorHasOccurred ? " nhsuk-u-padding-left-5" : "");
+  var emailCss = "nhsuk-checkboxes__conditional"
+                 + (Model.ReceiveNotificationEmails ? "" : " nhsuk-checkboxes__conditional--hidden")
+                 + (errorHasOccurred ? " nhsuk-u-padding-left-5" : "");

```

Comment:
Copied this from another place on the site where we implement conditionally revealing checkboxes. It seems the alignment of the red line when there is an error is slightly to the left of the words above, it's barely noticeable but this fixes it. See below, the second checkbox has the padding removed: 
![image](https://user-images.githubusercontent.com/70278044/140484421-7706510a-08d3-4d92-ad4c-5b5271f946e8.png)

Reply:
OK I guess that's fine, it barely makes a difference.

Reply:
Looking at this again, I don't think the logic is right, unless it is intended to always add padding despite the error being on a different property.

Reply:
You're right, the logic is off. Since it's a microscopic change, probably not worth spending more time on it, I've removed the class. But let me know if you think otherwise.

Reply:
Agreed, for the sake of moving a line something like 10px I don't think it's worth changing it further.